scripts/CNN/CNN_Analysis.py

```diff
@@ -38,8 +38,8 @@
 plt.xlabel('Target')
 plt.ylabel('Output of Network')
 plt.title('Output of Pytorch-trained Network for Benchmark-Set')
```

Comment:
This xlim and ylim are just proposals. Usually it make sense, because the data only have points between 2 and 12.